src/PyCall.jl

```diff
@@ -979,6 +979,14 @@ include("serialize.jl")
 
 include("pyinit.jl")
 
+function Base.deepcopy_internal(obj::PyObject, stackdict::Base.IdDict)
+    msg = """
+    Deepcopy is disabled on PyObject. See
+    https://github.com/JuliaPy/PyCall.jl/issues/757
+    """
+    error(msg)
+end
```

Comment:
```suggestion
const _deepcopy = PyNULL()

function Base.deepcopy_internal(obj::PyObject, stackdict::Base.IdDict)
    haskey(stackdict, obj) && stackdict[obj]
    ispynull(_deepcopy) && copy!(_deepcopy, pyimport("copy")["deepcopy"])
    return pycall(_deepcopy, PyObject, obj)
end
```

Reply:
Or alternatively the following implementation is more conservative:
```jl
function Base.deepcopy_internal(obj::PyObject, stackdict::Base.IdDict)
    haskey(stackdict, obj) && stackdict[obj]
    return pyincref(obj)
end
```

Reply:
Personally I expect that I can do whatever mutation I want with a deepcopy and it does never any effect on the original object. So from that perapective the refinc is a footgun. Too much copy is the lesser evil in my opinion. The third option is to throw an error and force the user to think how to best deal with the given case, which can be annoying. Probably your copy.deepcopy solution is the best option.